binding.gyp

```diff
@@ -35,6 +35,9 @@
 							'-Wl,-rpath,<(opengl_bin)',
 							'<(opengl_bin)/glfw.dylib',
 						],
+						'defines': [
+							'__APPLE__'
```

Comment:
When I was compiling on my OSX machine I wasn't seeing `__APPLE__` automatically defined. It could be misconfiguration on my end, I just have a minimal OSX install with Xcode.

Reply:
Ok, we may arrange that. Maybe it should be done for all platforms then?